Bottom: a63efbb7f5887bb53271295a7dd840c6b77dbe69
Top:    fcf41d019b13f39a064e9ff61cf6ac5a07109fdf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-06-09 12:36:25 +0100

Temp build fixes (NOT NECESSARILY RIGHT!)


---

diff --git a/src/hid/gtk/gui-drc-window.c b/src/hid/gtk/gui-drc-window.c
index fa2ec86..c9916d4 100644
--- a/src/hid/gtk/gui-drc-window.c
+++ b/src/hid/gtk/gui-drc-window.c
@@ -166,13 +166,13 @@ unset_found_flags (int AndDraw)
       }
   }
   ENDALL_LOOP;
-  COPPERPOLYGON_LOOP (PCB->Data);
+  COPPERPOUR_LOOP (PCB->Data);
   {
-    if (TEST_FLAG (flag, polygon))
+    if (TEST_FLAG (flag, pour))
       {
-	AddObjectToFlagUndoList (POLYGON_TYPE, layer, polygon, polygon);
-	CLEAR_FLAG (flag, polygon);
-	DrawPolygon (layer, polygon, 0);
+	AddObjectToFlagUndoList (POUR_TYPE, layer, pour, pour);
+	CLEAR_FLAG (flag, pour);
+	DrawPour (layer, pour, 0);
 	change = True;
       }
   }
